buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt

```diff
@@ -287,13 +290,27 @@ abstract class PublishingPlugin : Plugin<Project> {
               "or have a valid $RELEASE_CONFIG_FILE file at the root directory."
           )
         }
-        val libraryGroupProjects =
-          releasinglibraries.flatMap { it.projectsToRelease }.filterNotNull().toSet()
-        val releasingProjects = releasinglibraries.mapNotNull { it.project }.toSet()
-        if (!libraryGroupProjects.equals(releasingProjects)) {
+      }
+    }
+
+  /**
+   * Registers the [VALIDATE_LIBRARY_GROUPS_TO_PUBLISH_TASK] task.
+   *
+   * Validates that all library groups of all publishing projects are included in the release config
+   *
+   * @throws GradleException if a library is releasing without it's library group.
+   */
+  private fun registerValidateLibraryGroupsToPublishTask(
+    project: Project,
+    releasinglibraries: List<FirebaseLibraryExtension>
+  ) =
+    project.tasks.register(VALIDATE_LIBRARY_GROUPS_TO_PUBLISH_TASK) {
+      doLast {
+        val libraryGroupProjects = releasinglibraries.flatMap { it.librariesToRelease }
+        val missingProjects = libraryGroupProjects - releasinglibraries
+        if (missingProjects.isNotEmpty()) {
           throw GradleException(
-            "Some libraries in library groups are not in the release: " +
-              Sets.difference(libraryGroupProjects, releasingProjects).map { it.displayName }
+            "Some libraries in library groups are not in the release: ${missingProjects.map { it.mavenName }}"
```

Comment:
Without the `joinToString("\n")` this will likely print something like:
```
Some libraries in library groups are not in the release: ["com.google.firebase:firebase-annotations:16.2.0", ""com.google.firebase:firebase-components:17.1.0"]
```

Instead of:
```
Some libraries in library groups are not in the release: 
com.google.firebase:firebase-annotations:16.2.0
com.google.firebase:firebase-components:17.1.0
```
But it's not that big of a deal tbf.